sketch_map_tool/tasks.py

```diff
@@ -92,24 +93,40 @@ def georeference_sketch_maps(
     uuids: list[str],
     map_frames: dict[str, NDArray],
     bboxes: list[Bbox],
+    layers: list[Layer],
 ) -> AsyncResult | BytesIO:
-    def process(sketch_map_id: int, uuid: str, bbox: Bbox) -> BytesIO:
-        """Process a Sketch Map."""
+    def process(
+        sketch_map_id: int,
+        uuid: str,
+        bbox: Bbox,
+    ) -> BytesIO:
+        """Process a Sketch Map and its attribution."""
         # r = interim result
         r = db_client_celery.select_file(sketch_map_id)
         r = to_array(r)
         r = clip(r, map_frames[uuid])
         r = georeference(r, bbox)
         return r
 
+    def get_attribution_text(layer: Layer) -> BytesIO:
+        attribution = get_attribution(layer)
+        attribution = attribution.replace("<br />", "\n")
+        return BytesIO(attribution.encode())
+
     def zip_(file: BytesIO, file_name: str):
         with ZipFile(buffer, "a") as zip_file:
             name = ".".join(file_name.split(".")[:-1])
             zip_file.writestr(f"{name}.geotiff", file.read())
 
     buffer = BytesIO()
-    for file_id, uuid, bbox, file_name in zip(file_ids, uuids, bboxes, file_names):
+    for file_id, uuid, bbox, layer, file_name in zip(
+        file_ids, uuids, bboxes, layers, file_names
+    ):
         zip_(process(file_id, uuid, bbox), file_name)
+    for layer in layers:
+        with ZipFile(buffer, "a") as zip_file:
+            zip_file.writestr("attributions.txt", get_attribution_text(layer).read())
```

Comment:
I would move the iteration through the layers to `get_attribution_text`. -> Write just a single attributions.txt with all attributions in it. 
The way it is currently implemented, it would overwrite attibutions.txt in the case of multiple layers.